R/springer_article.R

```diff
@@ -0,0 +1,21 @@
+#' Springer Journal format.
+#'
+#' This format was adapted from the Springer Macro package for Springer
+#' Journals.
+#'
+#' @inheritParams rmarkdown::pdf_document
+#' @param ... Arguments to \code{rmarkdown::pdf_document}
+#' @return R Markdown output format to pass to \code{\link[rmarkdown:render]{render}}
+#' @examples
+#' \dontrun{
+#' rmarkdown::draft("MyArticle.Rmd", template = "springer_article", package = "rticles")
+#' }
+#'
+#' @export
+springer_article <- function(..., keep_tex = TRUE, citation_package = 'none'){
+  template <- find_resource("springer_article", "template.tex")
+  fmt <- inherit_pdf_document(template = template,
```

Comment:
I don't think it is necessary to create the variable `fmt` here. I mean the assignment is unnecessary.